cdap-docs/admin-manual/source/installation/installation.rst

```diff
@@ -281,7 +289,6 @@ Using Yum::
 
 Using APT::
 
-  sudo apt-get update
   sudo apt-get install cdap-gateway cdap-kafka cdap-master cdap-security cdap-web-app
```

Comment:
OK: this one: the line ``sudo apt-get update`` is still in the 2.8 installation. Should it have been/be removed?